dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/storage/GitHubJsonStore.java

```diff
@@ -13,8 +13,9 @@
 import java.io.InputStreamReader;
 import java.nio.file.Paths;
 import java.util.*;
+import java.util.stream.Collectors;
```

Comment:
Minor:  A nice feature in java reduce the terseness of code is a static import

instead of 
```
import java.util.stream.Collectors;

...

Collectors.toList()
```

You can do

```
import static java.util.stream.Collectors.toList;

...

toList()
```

This is a good pattern to follow for utility class methods especially in tests.